pkg/controller/template/render.go

```diff
@@ -207,6 +207,10 @@ func getPaths(config *RenderConfig, platformString string) []string {
 		platformBasedPaths = append(platformBasedPaths, sno)
 	}
 
+	// specific platform should be the last one in order
+	// to override on-prem files in case needed
+	platformBasedPaths = append(platformBasedPaths, platformString)
```

Comment:
To make extra sure we get it right this time, should this come before SNO? Such that SNO is the highest precedence?

Reply:
I'm not sure SNO is so trivial to handle... This is not a separate platform (today SNO is only `platform: none`), so how are we going (in the next release(s)) to handle vSphere SNO ?